openmetadata-ui/src/main/resources/ui/src/components/common/AsyncSelectList/AsyncSelectList.tsx

```diff
@@ -305,12 +306,13 @@ const AsyncSelectList: FC<AsyncSelectListProps & SelectProps> = ({
           />
         )
       }
+      open={open}
       optionLabelProp="label"
-      // this popupClassName class is used to identify the dropdown in the playwright tests
-      popupClassName="async-select-list-dropdown"
+      popupClassName="async-select-list-dropdown" // this popupClassName class is used to identify the dropdown in the playwright tests
       style={{ width: '100%' }}
       tagRender={customTagRender}
       onChange={handleChange}
+      onDropdownVisibleChange={setOpen}
```

Comment:
we are extending the props, you can pass Select props here. Can we use that ?
I think it was explicit decision from our end to not have open prop. Pls double check.